src/nanorc/common_commands.py

```diff
@@ -65,14 +65,13 @@ def ls(obj, legend):
 
 
 @click.command()
-@click.option('--pin-thread-file', type=click.Path(exists=True, resolve_path=True), default=None)
+@click.argument('pin-thread-file', type=click.Path(exists=True, resolve_path=True))
```

Comment:
Actually, this change is not backwards compatible, right? It means that a `pin_threads` command now _must_ have an argument of the thread pinning file, and won't default to what has been set in the environment. This may be ok, but would break instructions/workflows that have been given to others.

Reply:
Indeed, but if one has more than one thread pinning file, I'm not sure how I should choose it without specifying an argument here.